scripts/data_files/driver_templates/psa_crypto_driver_wrappers.h.jinja

```diff
@@ -2724,6 +2715,7 @@ static inline psa_status_t psa_driver_wrapper_pake_setup(
         case PSA_KEY_LOCATION_LOCAL_STORAGE:
             /* Key is stored in the slot in export representation, so
              * cycle through all known transparent accelerators */
+            status = PSA_ERROR_NOT_SUPPORTED;
```

Comment:
Looks like a dead store in some builds. Hope we don't create warnings for other compilers while fixing IAR warnings. The CI will tell us.